app/controllers/errors/InformationCannotBeAddedController.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.errors
+
+import config.AppConfig
+import controllers.actions._
+import javax.inject.{Inject, Singleton}
+import play.api.i18n.I18nSupport
+import play.api.mvc._
+import repositories.SessionRepository
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+import views.html.errors.InformationCannotBeAddedView
+
+import scala.concurrent.ExecutionContext
+
+@Singleton
+class InformationCannotBeAddedController @Inject()(identify: IdentifierAction,
+                                                   getData: DataRetrievalAction,
+                                                   requireData: DataRequiredAction,
+                                                   mcc: MessagesControllerComponents,
+                                                   sessionRepository: SessionRepository,
+                                                   view: InformationCannotBeAddedView,
+                                                   implicit val appConfig: AppConfig,
+                                                   implicit val ec: ExecutionContext)
+  extends FrontendController(mcc) with I18nSupport {
+
+  def onLoad(caseId: String): Action[AnyContent] = (identify andThen getData andThen requireData).async { implicit request =>
```

Comment:
Don't pass the case Id in as this will be a query parameter on the url. Look the case Id up using user answers in the controller before you delete the user answers.